### PR TITLE
Admin emails

### PR DESCRIPTION
**Pull request by nmohamed**
Sends emails to admins to verify and when verified email is sent to user

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -48,34 +48,51 @@ routes.POSTregister = function(req, res, next) {
...
 
-      email.addTo(user.email);
-      email.setFrom('SpringInitiative@olinjs.com');
-      email.setSubject('Confirm your email for Spring Initiative');
-      email.setHtml('<a target=_blank href=\"' + authenticationURL +
-        '\">Confirm \
-                  your email</a><br>This is an automated response. Do not reply'
-      );
-      sendgrid.send(email);
+        for (var i = admins.length - 1; i >= 0; i--) {
```
Comment (davidabrahams) [inline routes/index.js]: loop backwards literally what.

**Comment by aconsilvio** [comment]
:+1:

**Comment by BrennaManning** [comment]
:+1:

File: routes/index.js [hunk 2]
```diff
@@ -48,34 +48,51 @@ routes.POSTregister = function(req, res, next) {
...
+          email.addTo(admins[i].email);
+        };
+
+        email.setFrom('SpringInitiative@olinjs.com');
```
Comment (shrutiyer) [inline routes/index.js:63]: Have we created this email id yet?

File: routes/index.js [hunk 3]
```diff
@@ -48,34 +48,51 @@ routes.POSTregister = function(req, res, next) {
...
+        email.setSubject(user.email + ' wants to register for Spring Initiative\'s website.');
+        email.setHtml('<a target=_blank href=\"' + authenticationURL +
+          '\">Confirm their registration.</a>' + 
+          '<br>This is an automated response. Do not reply.');
+        sendgrid.send(email);
+
+        console.log("Email verification sent");
```
Comment (shrutiyer) [inline routes/index.js:70]: Console logs used for short-tern debugging can be removed I think

**Comment by shrutiyer** [comment]
I am not an admin. But if others have tested it :+1:

**Comment by davidabrahams** [comment]
Houston, we have a merge.

**Pull request merged**